src/test/ui/generator/unresolved_type_param.rs

```diff
@@ -0,0 +1,9 @@
+// edition:2018
```

Comment:
It'd be nice to have a comment here documenting the purpose of the test -- i.e., this test is checking the quality of error message produced when we have an unresolved type parameter etc.

Reply:
👍